apps/src/lib/client/tx.rs

```diff
@@ -420,6 +426,81 @@ pub async fn submit_transfer(ctx: Context, args: args::TxTransfer) {
     process_tx(ctx, &args.tx, tx, &keypair).await;
 }
 
+pub async fn submit_create_nft(ctx: Context, args: args::NftCreate) {
+    let file = File::open(&args.nft_data).expect("File must exist.");
+    let nft: Nft = serde_json::from_reader(file)
+        .expect("Couldn't deserialize nft data file");
+
+    let vp_code = match &nft.vp_path {
+        Some(path) => {
+            std::fs::read(path).expect("Expected a file at given code path")
+        }
+        None => ctx.read_wasm(VP_NFT),
+    };
+
+    let data = CreateNft {
+        tag: nft.tag.to_string(),
+        creator: nft.creator,
+        vp_code,
+        keys: nft.keys,
+        opt_keys: nft.opt_keys,
+        tokens: nft.tokens,
+    };
+
+    let data = data.try_to_vec().expect(
+        "Encoding transfer data to initialize a new account shouldn't fail",
+    );
+
+    let tx_code = std::fs::read(TX_CREATE_NFT)
+        .expect("Expected a file at given code path");
+
+    let tx = Tx::new(tx_code, Some(data));
+
+    let (ctx, tx, keypair) = sign_tx(ctx, tx, &args.tx, None).await;
+
+    process_tx(ctx, &args.tx, tx, &keypair).await;
+}
+
+pub async fn submit_mint_nft(ctx: Context, args: args::NftMint) {
+    let file = File::open(&args.nft_data).expect("File must exist.");
+    let nft_tokens: Vec<NftToken> =
+        serde_json::from_reader(file).expect("JSON was not well-formatted");
+
+    let nft_creator_key = nft::get_creator_key(&args.nft_address);
+    let client = HttpClient::new(args.tx.ledger_address.clone()).unwrap();
+    let nft_creator_address = match rpc::query_storage_value::<Address>(
+        client,
+        nft_creator_key,
+    )
+    .await
+    {
+        Some(addr) => addr,
+        None => {
+            eprintln!("No creator key found for {}", &args.nft_address);
+            safe_exit(1);
+        }
+    };
+
```

Comment:
Similar to nft init - 1/2:
```suggestion
let signer = Some(WalletAddress::new(nft_creator_address.to_string()));
```